### PR TITLE
Modify workflows and Ballerina.toml to release with Swan Lake GA

### PR DESCRIPTION
**Pull request by kaneeldias**
## Purpose
$subject

### CODE REVIEW

File: .github/workflows/central-publish.yml [hunk 1]
```diff
@@ -20,20 +20,20 @@ jobs:
       - uses: actions/checkout@v2
 
       - name: Ballerina Build
-        uses: ballerina-platform/ballerina-action/@slbeta5
+        uses: ballerina-platform/ballerina-action/@2201.0.0
         with:
           args:
-            build --skip-tests
+            build
```
Comment (niveathika) [inline .github/workflows/central-publish.yml]: Shouldn't this be pack now?

Comment (kaneeldias) [reply .github/workflows/central-publish.yml]: Removed in [59e3755](https://github.com/ballerina-platform/module-ballerinax-oracledb.driver/pull/3/commits/59e37552bf62ad713802822747419db74a5eb4d7)

Comment (niveathika) [reply .github/workflows/central-publish.yml]: We cant remove build command altogeather. AFAIK pack does not run test cases

Comment (niveathika) [reply .github/workflows/central-publish.yml]: Lets have the test command instead of build. Atleast as commented step

**Review by niveathika — COMMENTED** [review]
Lets update the documentation with distribution change

**Comment by kaneeldias** [comment]
> Lets update the documentation with distribution change

@niveathika the documentation has been updated with [bdc9c45](https://github.com/ballerina-platform/module-ballerinax-oracledb.driver/pull/3/commits/bdc9c45d3363947be6a534138a6dd5c29eabe211)

**Review by niveathika — APPROVED** [review]

**Pull request merged**